cg/meta/workflow/raredisease.py

```diff
@@ -176,6 +177,7 @@ def get_workflow_metrics(self, sample_id: str) -> dict:
                 self.get_metric_conditions_by_prep_category(sample_id=sample.internal_id)
             )
             self.set_order_sex_for_sample(sample, metric_conditions)
+            self.set_adapter_bases_for_sample(sample, metric_conditions)
```

Comment:
when functions have more than 1 argument we use named args
```suggestion
            self.set_order_sex_for_sample(sample=sample, metric_conditions=metric_conditions)
            self.set_adapter_bases_for_sample(sample=sample, metric_conditions=metric_conditions)
```